guard/src/commands/mod.rs

```diff
@@ -23,30 +23,30 @@ pub const RULEGEN: &str = "rulegen";
 pub const TEST: &str = "test";
 pub const VALIDATE: &str = "validate";
 // Arguments for validate
-pub const ALPHABETICAL: (&str, &str) = ("alphabetical", "a");
-pub const DATA: (&str, &str) = ("data", "d");
-pub const LAST_MODIFIED: (&str, &str) = ("last-modified", "m");
-pub const OUTPUT_FORMAT: (&str, &str) = ("output-format", "o");
-pub const INPUT_PARAMETERS: (&str, &str) = ("input-parameters", "i");
-pub const PAYLOAD: (&str, &str) = ("payload", "P");
-pub const PREVIOUS_ENGINE: (&str, &str) = ("previous-engine", "E");
-pub const PRINT_JSON: (&str, &str) = ("print-json", "p");
-pub const SHOW_CLAUSE_FAILURES: (&str, &str) = ("show-clause-failures", "s");
-pub const SHOW_SUMMARY: (&str, &str) = ("show-summary", "S");
-pub const TYPE: (&str, &str) = ("type", "t");
-pub const VERBOSE: (&str, &str) = ("verbose", "v");
+pub const ALPHABETICAL: (&str, char) = ("alphabetical", 'a');
```

Comment:
Nice catch! Should have had `char` in here in the first place since `args.short()` only accepts character.